src/tokens/TokenBalances.tsx

```diff
@@ -98,9 +111,29 @@ function TokenBalancesScreen({ navigation }: Props) {
     )
   }
 
+  const onPressNFTsBanner = () => {
+    ValoraAnalytics.track(HomeEvents.view_nft_home_assets)
+    navigation.navigate(Screens.WebViewScreen, {
+      uri: `https://nfts.valoraapp.com/?address=${walletAddress}&hide-header=true`,
+    })
+  }
+
   return (
     <>
-      {showPriceChangeIndicatorInBalances && (
+      {shouldVisualizeNFTsInHomeAssetsPage && (
+        <Touchable borderless={true} onPress={onPressNFTsBanner}>
+          <View style={[styles.bannerContainer]}>
```

Comment:
i think the styles can be simplified here a bit :)

the banner container can provide the padding already for all its children (e.g. `paddingHorizontal: Spacing.Thick24, paddingVertical: ... `) and use `justifyContent: 'space-between'` to push its children to the sides. with this, you don't need to have a left container anymore, and the right container probably doesn't need to have styles other than `flexDirection: 'row'`. what do you think?

Reply:
Wow, you're right. I just changed the styles according to your comments. https://github.com/valora-inc/wallet/pull/2632/commits/c84ea0140d177194851ed218219433dc32f54e2a
This way was what I was looking for when I implemented it. Thanks for letting me know!